security_monkey/task_scheduler/beat.py

```diff
@@ -37,8 +37,23 @@ def setup_the_tasks(sender, **kwargs):
             app.logger.info("[ ] Scheduling tasks for {type} account: {name}".format(type=account.type.name,
                                                                                      name=account.name))
             rep = Reporter(account=account.name)
+
+            # Is this a dedicated watcher stack, or is this stack ignoring anything?
+            only_watch = get_sm_celery_config("security_monkey_watcher_ignore", set)
+            # If only_watch is set, then ignoring is ignored.
+            if only_watch:
+                ignoring = set()
+            else:
+                # Check if we are ignoring any watchers:
+                ignoring = get_sm_celery_config("security_monkey_watcher_ignore", set) or set()
+
             for monitor in rep.all_monitors:
-                if monitor.watcher:
+                # Is this watcher enabled?
+                if monitor.watcher and monitor.watcher not in ignoring:
```

Comment:
does this not have to be `monitor.watcher.index` not in ignoring?

Reply:
Good catch.